packages/office-ui-fabric-react/src/components/ComboBox/ComboBox.tsx

```diff
@@ -1040,7 +1043,7 @@ export class ComboBox extends BaseComponent<IComboBoxProps, IComboBoxState> {
           onMouseLeave={ this._onOptionMouseLeave }
           role='option'
           aria-selected={ isSelected ? 'true' : 'false' }
-          ariaLabel={ item.text }
+          ariaLabel={ this._getPreviewText(item) }
```

Comment:
What about the multiSelect case?

Reply:
I'm not sure what you mean..

Reply:
the checkbox below seems to be missing the ariaLabel... we should fix that